pkgs/servers/demoit/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, buildGoPackage
+, fetchFromGitHub
+}:
+
+buildGoPackage rec {
+  name = "demoit-${version}";
+  version = "20190215";
+  goPackagePath = "github.com/dgageot/demoit";
+
+  src = fetchFromGitHub {
+    owner = "dgageot";
+    repo = "demoit";
+    rev = "391ed33bfb6b459689e0f879dcf071bb4eee1608";
+    sha256 = "13jk1r4xillcwf4biqr0cyvdvm62kg808pq8fxlqsd47nw3l8pp1";
```

Comment:
Tried to build, saw this:
```
hash mismatch in fixed-output derivation '/nix/store/swzqsc7yrwair0dsvfx6nkf5dfkdgp45-source':
  wanted: sha256:13jk1r4xillcwf4biqr0cyvdvm62kg808pq8fxlqsd47nw3l8pp1
  got:    sha256:03h0ad3v9xfigrppbcxiykzgcrv3dnarm7m1vixi2j4n1340f7q6
```

Reply:
strange behavior, I updated to latest commit

Reply:
@freezeboy how are you getting your sha256 hashes? For the new revision you listed you should run this: `nix-prefetch-git --url https://github.com/dgageot/demoit.git --rev ec70fbdf5a5e92fa1c06d8f039f7d388e0237ba2` which gives `01584cxlnrc928sw7ldmi0sm7gixmwnawy3c5hd79rqkw8r0gbk0`.

Reply:
Well I used the suggestion with the conflict ... anyway, I forced-pushed a fix and I checked it builds.